### PR TITLE
Break up and move Best Practices topic.

### PR DESCRIPTION
**Pull request by JStickler**
I realized that I should have moved the Best Practices topic as part of #8711, since most of the content is best practices for using labels.
This PR:
* Splits the file into two parts (Best practices for labels and best practices for configuration)
* Moves the two files to Get Started and Configure, respectively.
* Updates page weights for both sections as needed.

### CODE REVIEW

File: docs/sources/configure/bp-configure.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Configuration best practices
+menuTitle:  Best practices
+description: Grafana Loki configuration best practices
+weight:  100
+---
+# Configuration best practices
+
+Grafana Loki is under active development, and we are constantly working to improve performance. But here are some of the most current best practices for configuration that will give you the best experience with Loki.
```
Comment (trevorwhitney) [inline docs/sources/configure/bp-configure.md:9]: should we qualify "current" with a version?

File: docs/sources/configure/bp-configure.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+{job="syslog", instance="host2"} 00:00:02 i'm a syslog!  <- Accepted, still in order for stream 2
+```
+
+But what if the application itself generated logs that were out of order? Well, I'm afraid this is a problem. If you are extracting the timestamp from the log line with something like [the Promtail pipeline stage](/docs/loki/latest/clients/promtail/stages/timestamp/), you could instead _not_ do this and let Promtail assign a timestamp to the log lines. Or you can hopefully fix it in the application itself.
+
+It's also worth noting that the batching nature of the Loki push API can lead to some instances of out of order errors being received which are really false positives. (Perhaps a batch partially succeeded and was present; or anything that previously succeeded would return an out of order entry; or anything new would be accepted.)
+
+## Use `chunk_target_size`
+
+Using `chunk_target_size` instructs Loki to try to fill all chunks to a target _compressed_ size of 1.5MB. These larger chunks are more efficient for Loki to process.
```
Comment (trevorwhitney) [inline docs/sources/configure/bp-configure.md:55]: the 1.5MB is based on what the size is configured to. This value is in bytes.

Are we just saying 1.5MB is the default?

File: docs/sources/get-started/bp-labels.md [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
 description: Grafana Loki label best practices
-weight: 400
+aliases: 
+- ../best-practices/
+weight: 700
 ---
-# Best practices
+# Label best practices
 
 Grafana Loki is under active development, and we are constantly working to improve performance. But here are some of the most current best practices for labels that will give you the best experience with Loki.
```
Comment (trevorwhitney) [inline docs/sources/get-started/bp-labels.md:11]: this paragraph is repeated verbatim in the added file above

**Review by trevorwhitney — APPROVED** [review]
approving to unblock, but I have a few bits of feedback. thanks!

**Review by MichelHollands — APPROVED** [review]

File: docs/sources/configure/bp-configure.md (hunk 1 above)
Comment (JStickler) [reply docs/sources/configure/bp-configure.md:9]: Do we have any idea what version of Loki these best practices were written against?

File: docs/sources/get-started/bp-labels.md (hunk 3 above)
Comment (JStickler) [reply docs/sources/get-started/bp-labels.md:11]: That's by design.

File: docs/sources/configure/bp-configure.md (hunk 2 above)
Comment (JStickler) [reply docs/sources/configure/bp-configure.md:55]: ¯\_(ツ)_/¯  I don't know, I didn't write this content, I'm just moving it around in this PR.

**Pull request merged**